security/src/main/java/no/difi/meldingsutveksling/IntegrasjonspunktNokkel.java

```diff
@@ -14,6 +14,11 @@
  */
 public class IntegrasjonspunktNokkel {
 
+    private static final String ERR__MISSING_PRIVATE_KEY_OR_PASS = "Problem accessing PrivateKey with alias \"%s\" inadequate access or Password is wrong";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00115)